devtools/server/actors/replay/telemetry.js

```diff
@@ -11,36 +11,25 @@ function pingTelemetry(source, name, data) {
 
   if (!enabled || !url) return;
 
-  // fetch the user info to send in the `browserUser` field.
-  let browserUser = "unknown";
-  if (ReplayAuth.hasOriginalApiKey()) {
-    browserUser = "fixed-api-key";
-  } else {
-    const token = ReplayAuth.getReplayUserToken();
-    const tokenInfo = token ? ReplayAuth.tokenInfo(token) : null;
-    if (tokenInfo) {
-      browserUser = tokenInfo.payload.sub || "no-user-field";
-    } else if (token) {
-      browserUser = "invalid-json";
-    } else {
-      browserUser = "no-user-token";
-    }
-  }
+  // fetch the user info to send in `Authorization` header.
+  const auth = ReplayAuth.getOriginalApiKey() || ReplayAuth.getReplayUserToken();
 
   // Collect info to send for `browserSettings` field.
   const usePreallocated = Services.prefs.getBoolPref("devtools.recordreplay.usePreallocated");
   const browserSettings = { usePreallocated };
 
   fetch(url, {
     method: 'POST',
+    headers: {
+      Authorization: `Bearer ${auth}`,
```

Comment:
We avoid sending `undefined` or `null` if there is no token, so maybe like 
```
headers: auth ? { Authorization: `Bearer ${auth}` } : undefined
```
?